main.go

```
@@ -344,20 +320,17 @@ func applyTemplate(name string, text string, data template.Data) (string, error)
 	return result.String(), nil
 }
 
-func validateIncident(incident Incident) error {
-	impact := incident["impact"].(string)
-	if len(impact) > 0 {
-		if _, err := strconv.Atoi(impact); err != nil {
+func validateIncident(incident Incident) {
```

I think this should return an error anyways wether we proceed with incident creation or not